docs/docs/testcerts.md

```diff
@@ -0,0 +1,46 @@
+# Test Certificates
+
+To be able to start the SaltyRTC server, you need to specify a TLS key and
+certificate. In production you will want to use a certificate signed by a
+trusted CA, but for testing purposes, the easiest way is to create a
+self-signed certificate.
+
+## Generating a Test Certificate
+
+Use the following command to create such a certificate, valid for `localhost`
+during the next 5 years:
+
+    $ openssl req \
+           -newkey rsa:1024 \
```

Comment:
I'd say 4096. Sure, it's a test certificate but you know... people do stupid things and I don't want to be responsible. :wink:

Reply:
The certificate is only valid for `localhost`. I don't think there's much harm that can be caused by that. Do you think there's any value in >1024?

Reply:
I was thinking of people that generate production certs and are looking for examples.